ozcar-theia/.htaccess

```diff
@@ -18,19 +18,19 @@ RewriteCond %{HTTP_ACCEPT} !text/turtle
 RewriteCond %{HTTP_ACCEPT} !application/json
 RewriteCond %{HTTP_ACCEPT} text/html [OR]
 RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml
-RewriteRule ^(.+) http://in-situ.theia-land.fr/vocabularies/Skosmos/theia_ozcar_thesaurus/page/$1  [R=303,NE,L]
+RewriteRule ^(.+) http://in-situ.theia-land.fr/vocabularies/Skosmos/theia_ozcar_thesaurus/page/?uri=https\%3A\%2F\%2Fw3id.org\%2F$0  [R=303,NE,L]
 
 # Rewrite rule to serve RDF/XML content from the vocabulary URI if requested
 RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
-RewriteRule ^(.+) http://in-situ.theia-land.fr/vocabularies/Skosmos/rest/v1/theia_ozcar_thesaurus/data?uri=https\%3A%\2F%\2Fw3id.org\%2Fozcar-theia\%2F$1&format=application/rdf\%2Bxml  [R=303,NE,L]
+RewriteRule ^(.+) http://in-situ.theia-land.fr/vocabularies/Skosmos/rest/v1/theia_ozcar_thesaurus/data?uri=https\%3A\%2F\%2Fw3id.org\%2F$0&format=application/rdf\%2Bxml  [R=303,NE,L]# Rewrite rule to serve TURTLE content from the vocabulary URI if requested
```

Comment:
There's a trailing comment here that's a copy of the below comment.  Same with next two changed lines.  Look like typos.

Reply:
Thanks. The typos has been corrected